src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java

```diff
@@ -1997,4 +2065,8 @@ private HyperwalletPayPalAccount copy(HyperwalletPayPalAccount payPalAccount) {
         return payPalAccount;
     }
 
+    private HyperwalletTransferRefund copy(HyperwalletTransferRefund transferRefund) {
+        transferRefund = HyperwalletJsonUtil.fromJson(HyperwalletJsonUtil.toJson(transferRefund), HyperwalletTransferRefund.class);
+        return transferRefund;
+    }
```

Comment:
Can be changed to: return HyperwalletJsonUtil.fromJson(HyperwalletJsonUtil.toJson(transferRefund), HyperwalletTransferRefund.class);

Reply:
Done